### PR TITLE
Update presubmission_inquiry.md

### PR DESCRIPTION
**Pull request by GreiffLab**
Improved flow of Cover letter, fixed typos

### CODE REVIEW

File: content/presubmission_inquiry.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
-Dear Dr. Markel:
+Dear Dr. Markel,
 
-I am writing to submit a Ten Simple Rules article entitled “Ten Simple Rules for
-Deep Learning in Biology.” Deep learning (DL) is exploding in popularity and is
-increasingly finding its way into biological data analysis. However, since most
-biologists receive little (or no) formal data science training, using deep learning
-properly remains a daunting task.
+it is with great enthusiasm that I am writing to submit a presubmission inquiry for a Ten Simple Rules article entitled “Ten Simple Rules for Deep Learning in Biology.” Deep learning (DL) is currenty exploding in popularity and is
```
Comment (julianstanley) [inline content/presubmission_inquiry.md]: "It is with". capitalize the first letter, right?

File: content/presubmission_inquiry.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
+increasingly finding its way into biological data analysis given its ability to detect high-dimensional, and for humans invisible, patterns in large-scale biological datasets. However, since most
+biologists and also many data scientists have received and still receive little (or no) formal training in DL, correct application of DL to biological data analysis remains a daunting task hindering DL-accelerated scientific progress and enhanced biological insight.
```
Comment (julianstanley) [inline content/presubmission_inquiry.md]: Too wordy, IMO. Would revise to: "biologists rarely receive formal training in DL and its correct application to biological data analysis remains a daunting task that hinders scientific progress and enhanced biological insight."

File: content/presubmission_inquiry.md [hunk 3]
```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
 1. Lorem ipsum
 1. Lorem ipsum
 
-These rules are chosen so that readers of varying experience levels will be able to gain from the article and range from the high level (Should I use deep learning for this problem?) to implementation best practices.
+These rules have been devised to effectively reach audiences of varying DL expertise readers ranging from high level guidance to implementation of best practices.
```
Comment (julianstanley) [inline content/presubmission_inquiry.md]: One typo, but I think the phrasing is a bit awk. Would revise to: "These rules range from high-level guidance to implementation of best practices and they have have been devised to effectively reach audiences of varying expertise"

File: content/presubmission_inquiry.md [hunk 4]
```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
...
+Upon notification that this manuscript will be suitable for submission to PLOS Computational Biology, I will further engage with the community to solicit feedback and contributions. All authors of the paper will meet the ICMJE authorship standards.
 
-By providing guidance on deep learning, powerful DL methods can be properly utilized by the community towards a deeper understanding of biology. Here, we aim to increase the number of biological researchers that take advanatage of DL methods (by making DL more accessible) and improve the overall the quality and reproducibility of the research (by helping researchers avoid common mistakes).
+By providing guidance on deep learning, powerful DL methods can be properly utilized by the biological community towards a deeper understanding of biology. Here, we aim to increase the number of biological researchers that take advanatage of DL methods (by making DL more accessible) and improve the overall quality and reproducibility of DL research by helping researchers avoid common pitfalls.
```
Comment (julianstanley) [inline content/presubmission_inquiry.md]: Some typos. Would revise to: 
"By providing guidance on deep learning, powerful DL methods can be more properly utilized by the biological community. Here, we aim to increase the accessibility of DL techniques to biology and thereby improve the overall quality and reproducibility of DL in the literature."

File: content/presubmission_inquiry.md [hunk 5]
```diff
@@ -1,29 +1,28 @@
-Dear Dr. Markel:
+Dear Dr. Markel,
 
-I am writing to submit a Ten Simple Rules article entitled “Ten Simple Rules for
```
Comment (agitter) [inline content/presubmission_inquiry.md]: This is a general comment unrelated to this specific pull request, but for other Manubot manuscripts we found it easiest to keep one sentence per line without any hard line wrapping.  That makes it easier to review the changes in the GitHub diff interface and comment on lines that changed.

@Benjamin-Lee what do you think?  Should we adopt that strategy here?

File: content/presubmission_inquiry.md (hunk 4 above)
Comment (evancofer) [reply content/presubmission_inquiry.md]: Should we also switch the use of "deep learning" to "DL" to make it more consistent?

File: content/presubmission_inquiry.md [hunk 6]
```diff
@@ -1,29 +1,28 @@
...
 is added when applied to biology. Inspired by [*Opportunities and obstacles for
 deep learning in biology and medicine*](https://doi.org/10.1098/rsif.2017.0387)
 and [*Ten Simple Rules for Writing a PLOS Ten Simple Rules
 Article*](https://doi.org/10.1371/journal.pcbi.1003858) (Rule 5: Collaborate), I propose
 to write the article using
 [Manubot](https://github.com/greenelab/manubot-rootstock), a collaborative
 manuscript authoring platform based on GitHub. By writing the manuscript in the
-open on GitHub (which is extensively used by those doing deep learning research)
-and soliciting input from the community, I hope that the article will be able to
+open on GitHub (which is extensively used by those actively performing deep learning research)
```
Comment (evancofer) [inline content/presubmission_inquiry.md]: This is another place where we switch between "deep learning" and "DL".

File: content/presubmission_inquiry.md (hunk 4 above)
Comment (evancofer) [reply content/presubmission_inquiry.md]: Should we also switch from "deep learning" to "DL" to make this more consistent?

File: content/presubmission_inquiry.md (hunk 5 above)
Comment (evancofer) [reply content/presubmission_inquiry.md]: Given #57 , I think the switch to 1 sentence per line makes sense here or in a separate PR.

Comment (agitter) [reply content/presubmission_inquiry.md]: Yes, it may make sense to merge this and then have separate PR that changes whitespace but not content.  That way the diff is easier to review.

Comment (evancofer) [reply content/presubmission_inquiry.md]: I can do that after this PR is reworked and merged.

File: content/presubmission_inquiry.md (hunk 4 above)
Comment (rasbt) [reply content/presubmission_inquiry.md]: Maybe i am to nit-picky but "biological researchers" sounds weird ;). Maybe we can just say "biologists" or "computational as well as experimental biologists"

File: content/presubmission_inquiry.md [hunk 7]
```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
 1. Lorem ipsum
```
Comment (rasbt) [inline content/presubmission_inquiry.md]: I guess this needs to be removed?

File: content/presubmission_inquiry.md [hunk 8]
```diff
@@ -1,29 +1,28 @@
...
+and soliciting input from the wider scientific community, I am convinced that this manuscript will
 provide actionable, DL-specific guidance for both new and experienced DL
 practitioners.
 
 
 To accomplish this goal, I have created a
 [repository](github.com/Benjamin-Lee/deep-rules) that will host the discussion
 and writing of the manuscript. In preparation for submitting this letter of
-inquiry, the following rules have been proposed:
+inquiry, the following 10 rules have been proposed:
```
Comment (rasbt) [inline content/presubmission_inquiry.md]: maybe also write "ten" as word, like in the title

File: content/presubmission_inquiry.md (hunk 7 above)
Comment (evancofer) [reply content/presubmission_inquiry.md]: We should probably put the rules into a separate pull request after we merge this one. It may be ideal to use some sort of template that pulls them from the `rules.md` file. Then, the rules here would be updated automatically. @agitter @dhimmel any suggestions?

Comment (dhimmel) [reply content/presubmission_inquiry.md]: This is a presubmission inquiry right? I don't see much value to have this automatically update. It's not part of the manuscript, so I don't think our templating system would work.

Comment (evancofer) [reply content/presubmission_inquiry.md]: Souds good. This is probably worth committing an entire PR to. It is also slightly beyond the scope of the current PR (i.e. addressing general flow of the presubmission inquiry).

**Comment by dhimmel** [comment]
> julianstanley requested review from Benjamin-Lee, agitter, evancofer, sgfin, dhimmel and rasbt 5 hours ago

@julianstanley I'd like to take a more back seat role on this project. Happy to help with any issues related to Manubot and how to use it effectively. When it comes to actual content issues, I won't be of much help.
